script/src/proof_backends/sp1.rs

```diff
@@ -121,4 +123,8 @@ impl ProofBackend for SP1Backend {
 
         Ok(output)
     }
+
+    fn vkey_digest(&self) -> Vec<u8> {
```

Comment:
wdyt about returning `self.proving_key.vk.bytes32_raw()` instead so that we can use the resultant `[u8; 32]` to make an alloy `FixedBytes<32>`. At least this way we get better type checking in `VkeyResponse` (assuming that's important). We also don't need to use format! then.

Reply:
Sure, I think it's fine. I'm not that opinionated.